Tests/FoundationTests/DateExtensionsTests.swift

```diff
@@ -558,12 +558,12 @@ final class DateExtensionsTests: XCTestCase {
 		
 		date = Date()
```

Comment:
It will be better to initialize this based on a hardcoded value, not to do this again next year :)

Reply:
Initialized hardcoded date - 01.01.2018 :)